.circleci/config.yml

```diff
@@ -1,16 +1,16 @@
 version: 2
 jobs:
-  test:
+  complete-specs:
```

Comment:
This name is not very descriptive: All (complete) spec tests are run by the version-specific jobs as well, but this one runs other stuff in addition to spec (style check, etc... not sure exactly what else).

Reply:
What about `full-suite`?

Reply:
👍